### PR TITLE
Added ImageDimensionBody to ontologies-mico

### PR DESCRIPTION
**Pull request by thomas-idmt**
Small commit, much description. But imo, it highlights some workflow issues. Or at least issues of me understanding the overall architecture inside my 32°C office sauna ;-)

So, please comment :)
### Starting Point

The information about the dimensions (pixel) of an annotated image wasn't stored as rdf, but I needed it to calculate the "coverage" of annotations on the picture. It was available in the annotator's xml output, so i thought _Let's create an annotation for that!_ ;-)
### Assumption 1

This can be solved easily by modifying the object detection-to-rdf extractor and it is the right place for this annotation. Like this:
[ObjectToRDF.java:105](https://bitbucket.org/mico-project/extractors/commits/5d03a7a60db69993d6cbea49f53329c2e8651f59?at=master#Ljava/TE-202-object-detection-To-RDF/src/main/java/de/fhg/idmt/mico/extractors/or/ObjectToRDF.javaT105)

I have the suspicion that I might have chosen the wrong target, but it did what it should. However - the question was: where to put the data? 
So..
### Assumption 2

Every time I want to store something "new" inside the platform using Anno4J, I have to create a new Body class that is best stored in ontologies-mico.

(aside: there might be already something existing that is capable of storing two dimensional values, but it might have been something completely different where a dedicated class is needed).
And this is where this new body-class comes in. But sadly, it introduces a new dependency between the extractor and anno4j which wasn't there before.

On the other hand, on the querying side - it is needed again:
[Anno4JTest.java:81](https://bitbucket.org/mico-project/recommendation/commits/c44d584fb3b25763bcfe46977286f7621e1eb618#LDataCollector/src/test/java/Anno4JTest.javaT81)

so everything is fine (?) , with the exception that I have to wait for a new Anno4j-Release with the proposed changes, before committing my changes to the extractor. (which is not really bad, but puts some pressure on your release cycle).

Long story short: is this how it's supposed to work or am I completely off the track?

### CODE REVIEW

File: anno4j-ontologies/ontologies-mico/src/main/java/eu/mico/platform/anno4j/model/impl/body/ImageDimensionBody.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    @Iri(MICO.HAS_CONFIDENCE)
+    private Double confidence;
+
+    @Iri(MICO.HAS_WIDTH)
+    private Long width;
+
+    @Iri(MICO.HAS_HEIGHT)
+    private Long height;
+
+
```
Comment (schlegel) [inline anno4j-ontologies/ontologies-mico/src/main/java/eu/mico/platform/anno4j/model/impl/body/ImageDimensionBody.java:32]: side note: We always support a standard constructor in case somebody implements a custom constructor in future. A standard constructor is always needed for anno4j RDF mapping.

**Comment by schlegel** [comment]
in this special case width and height makes sense on the body side, because they are real descriptions about the media item. in model version 2 these media informations will be directly attached to the initial resource

**Comment by schlegel** [comment]
we also added a comment in ObjectToRDF.java:105. the contentpart was wrong
